tests/test_user_registration.py

```diff
@@ -0,0 +1,79 @@
+"""
+This file holds user registration test cases.
+"""
+import unittest
+import json
+from api import create_app
+
+
+class AuthTestCase(unittest.TestCase):
+    """
+        Test case for User registration.
+    """
+
+    def setUp(self):
+        """Set up test variables."""
+        self.app = create_app(config_name="testing")
```

Comment:
Is this test for challenge 2 or challenge 3?

Reply:
This one is for challenge 3, but from our talk on Monday, you told me to write the user tests which I was supposed to deliver by 8 pm the same day.